docs/getting-started/README.md

```diff
@@ -4,39 +4,46 @@ order: 1
 
 # Getting Started
 
+This document will guide you through setting up Kestra in a few minutes.
 
-## Before you begin
-Make sure the followings are already installed:
-- [Docker](https://docs.docker.com/engine/install/)
-- [Docker Compose](https://docs.docker.com/compose/install/)
+## Prerequisites
 
-## Launch Kestra
+Make sure the following is installed:
+- [Docker Desktop](https://docs.docker.com/get-docker/)
```

Comment:
Docker desktop is not needed only Docker is. This should be clearly stated that desktop is to makes things easier

Reply:
I have omitted that in favor of brevity. Is the alternative Docker Engine?

Reply:
The alternative is just Docker. On Linux at least.
On non Linux system there is a lot of alternatives